engine/api/src/main/scala/pl/touk/nussknacker/engine/api/JobData.scala

```diff
@@ -1,3 +1,5 @@
 package pl.touk.nussknacker.engine.api
 
-case class JobData(metaData: MetaData, processVersion: ProcessVersion)
+import pl.touk.nussknacker.engine.api.deployment.DeploymentVersion
+
+case class JobData(metaData: MetaData, processVersion: ProcessVersion, deploymentVersion: DeploymentVersion)
```

Comment:
I wonder if it is a good time to start using more human readable name for this data. It doesn't look like a version at all ;) Or at least add here to todo that it should be renamed...

Reply:
:+1: I changed to DeploymentData